pom.xml

```diff
@@ -121,36 +121,16 @@
     </dependency>
     <!-- jenkins dependencies -->
     <!-- test dependencies -->
-    <dependency>
-      <groupId>org.jenkins-ci.plugins.workflow</groupId>
-      <artifactId>workflow-api</artifactId>
-      <scope>test</scope>
-    </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-job</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.plugins.workflow</groupId>
-      <artifactId>workflow-basic-steps</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.plugins.workflow</groupId>
-      <artifactId>workflow-durable-task-step</artifactId>
```

Comment:
And this.

(Unless both are getting picked up from `docker-workflow`?)

Reply:
Tests are compiling fine and passing, why would I need this then?

Reply:
Probably it is getting picked up from `docker-workflow` then.